{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+class AccountConfig(AppConfig):
+	name = "account"
```

Comment:
Was able to build (./test.sh) and run the server within the created project without an issue